Digital_Fusion/src/components/Main/Main.jsx

```diff
@@ -0,0 +1,47 @@
+import "../Main/Main.css";
+import React from "react";
+import { useEffect, useState } from "react";
+import headerVideo from "../../assets/HeaderVideo.mp4";
+import { CardContainer } from "../CardContainer/CardContainer.jsx";
+export function Main() {
+  const [isScrolled, setIsScrolled] = useState(false);
+
+  const checkScroll = () => {
+    if (window.scrollY > 40) {
+      setIsScrolled(true);
+    } else {
+      setIsScrolled(false);
+    }
+  };
+  useEffect(() => {
+    checkScroll();
+
+    const interval = setInterval(checkScroll, 100);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  return (
+    <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component